### DISCUSSION TITLE
Can you guide me on certificate installation steps?

### DISCUSSION THREAD

**Discussion by YogarajRethinasamy** [International-Data-Spaces-Association/DataspaceConnector#90] [Q&A] [answered]
I have placed certificate in conf folder & changed the below thing in config.json files as per docs

IF i choose  idsc:PRODUCTIVE_DEPLOYMENT & rebuilt it again via ./maven clean package 

it throws me the following error 

[ERROR] Errors:
[ERROR]   ContractAgreementHandlerTest.handleMessage_validAgreement_returnMessageProcessedNotificationMessageAndLogToCH:248 ClassCast
[ERROR]   ContractRequestHandlerTest.checkContractRequest_validRequest_returnOk:493 ClassCast
[ERROR]   DescriptionRequestHandlerTest.handleMessage_validResourceDescriptionMsgKnownId_returnResourceDescription:176 ClassCast
[ERROR]   DescriptionRequestHandlerTest.handleMessage_validSelfDescriptionMsg_returnSelfDescription:114 ClassCast
[INFO]
[ERROR] Tests run: 1341, Failures: 0, Errors: 4, Skipped: 1

Rest of my changes are working fine if i choose test deployment build has been success

**Comment by ronjaquensel** [comment, marked as answer]
Duplicate of [#598](https://github.com/International-Data-Spaces-Association/DataspaceConnector/discussions/598)
